mix.exs

```diff
@@ -41,9 +45,9 @@ defmodule OpentelemetryEcto.MixProject do
   defp deps do
     [
       {:telemetry, "~> 0.4.2"},
-      {:opentelemetry_api, "~> 1.0.0-rc", allow_pre: true},
-      {:opentelemetry, "~> 1.0.0-rc", allow_pre: true},
-      {:ex_doc, "~> 0.21.0", only: [:dev], runtime: false},
+      {:opentelemetry_api, "~> 1.0.0-rc"},
```

Comment:
Neat, so this will pick up any `rc` release? I need to see if this works in rebar3.. that'd be nice.

Reply:
Yeah. I was wrong about the allow_pre thing. That was apparently a thing way back in Elixir. It's not necessary anymore. Just need the `-rc` part. It works for `-beta` or `-alpha` as well.